wp-content/plugins/hurumap/src/ChartDefinition/HurumapChart.js

```diff
@@ -104,25 +105,37 @@ const dataAggregateOptions = [
     label: 'Raw',
     value: 'raw'
   },
+  {
+    label: 'Raw (Percent)',
+    value: 'raw:percent'
```

Comment:
QQ: Is there a difference between `raw:percent` and just `:percent`?

Reply:
No difference
Since `value: 'raw'` then the percent will be `value: 'raw:percent'`. Just sticking with the format. @kilemensi